src/main/java/com/elearningbackend/utility/Constants.java

```diff
@@ -0,0 +1,7 @@
+package com.elearningbackend.utility;
+
+public abstract class Constants {
+    public static final int ZERO = 0;
```

Comment:
should use interface instead of abstract class. When it is interface, all vars are "public static final" as default